### ISSUE TITLE
Enhancement Request

### ISSUE THREAD

**Issue by dk2112** [Microsoft/winfile#210]
First, thanks for bringing back the best file management tool that ever existed.  A (should be easy) enhancement that I would like to see is an option to IGNORE ERRORS encountered while copying/moving/deleting files.  I (and likely others) use File Manager a lot as a recovery/migration tool for working with 'troubled' machines.  The goal is to reclaim as much as possible from the source.  As-is, you get a popup for every troublesome file and have to OK them one at a time.

**Comment by clzls** [comment]
Agree. Or maybe a `Ignore All` button on popup windows to temporary ignore errors.

**Comment by dk2112** [comment]
Either would be great.  I've loyally used winfile for years.  On the 
recent Windows versions, you had to use a hack called wfrevive that 
allowed it to work.  But, even with that, the original code was so old 
that it didn't understand distributed file systems or long directory 
paths.

Thanks for the consideration.  I appreciate all of the effort that it 
took to update the source code and re-release it.

------ Original Message ------

**Comment by Amaroq-Clearwater** [comment]
I would also appreciate these changes. However, you should change the title of this issue to include a summary of the requested feature.
